.gitignore

```diff
@@ -1,3 +1,5 @@
+.idea
```

Comment:
Directory `.idea` should be in your home dirirectory `.gitignore`.

Reply:
Done